resources/kcp/charts/component-reconcilers/templates/policy.yaml

```
@@ -0,0 +1,18 @@
+{{- range $component := .Values.global.components }}
+{{ if $.Values.serviceMonitor.enabled }}
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  labels:
+    component: {{ $component }}
+  name: {{ $component }}-reconciler
+  namespace: {{ $.Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      component: {{ $component }}
+  mtls:
+    mode: PERMISSIVE
```

Would it make sense to restrict all workloads with the matched label on a specific (http) port?
```suggestion
  mtls:
    mode: STRICT
  portLevelMtls:
    8080:
      mode: PERMISSIVE
```

---

cool, don't know this feature, that's totally valid point.